### DISCUSSION TITLE
Should a normal Yoda user be able to overwrite the metadata "ORG_REVISION_SCHEDULED"

### DISCUSSION THREAD

**Discussion by chStaiger** [UtrechtUniversity/yoda#221] [Q&A] [answered]
I as a normal user of the YODA system can overwrite the value of the metadata with the key ORG_REVISION_SCHEDULED. What are the implications of that? Especially when the value is set to some nonsense string?

**Comment by lwesterhof** [comment, marked as answer]
This is currently how it is supposed to work in the research space. With later iRODS releases (v4.3 or later) we will get the possibility to secure this metadata in the research space. When the value is set to some nonsense string the revision will fail.
